### ISSUE TITLE
cannot import name 'NaN' from 'numpy' 

### ISSUE THREAD

**Issue by johanatan** [OpenBB-finance/OpenBB#6555]
Following the tutorials to install the platform and then the cli, using condo et al, successfully on an M3 macbook air and then getting the following traceback upon trying to run 'openbb' from the condo openbb environment:

```
  File "/usr/local/Caskroom/miniconda/base/envs/openbb/lib/python3.11/site-packages/pandas_ta/momentum/squeeze_pro.py", line 2, in <module>
    from numpy import NaN as npNaN
ImportError: cannot import name 'NaN' from 'numpy' (/usr/local/Caskroom/miniconda/base/envs/openbb/lib/python3.11/site-packages/numpy/__init__.py)
```

**Comment by deeleeramone** [comment]
Hi there, you can resolve this error by downgrading the version of Numpy to one in the specs.

```
pip install numpy==1.26.4
```

**Comment by johanatan** [comment]
Ok thanks. Just curious why if this is needed, it isn't already set in the requirements.txt ?

**Comment by deeleeramone** [comment]
Good question. Numpy gets installed by Pandas and the issue arises from syntax in the `pandas_ta` library because the version is not pinned there, it just lets the version installed by Pandas live on. In the next release, this will be pinned on our side so that it doesn't continue to be an issue.

**Comment by johanatan** [comment]
Ok, good to know. Thanks

**Comment by deeleeramone** [comment]
This is resolved with #6572
